mk_verificator/utils/soft_assert.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import with_statement
```

Comment:
TODO (msenin): Add example how to use
TODO (msenin): Add/Change logic for `except Exception:` block

And discuss approach of usage this module with teammates